app/mailers/welcome_email_mailer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class WelcomeEmailMailer < GovukNotifyRails::Mailer
+  def send_welcome_email(first_name:, email:)
```

Comment:
Do we need the the `_welcome_email` here. Seems redundant. Also this creates it rather than sends it. Maybe `def mail` and then you get `WelcomeEmailMailer.mail.deliver_later` or something like that?

Reply:
Renamed to `generate`